src/components/AddressResolver/EndpointEntry/EndpointEntry.tsx

```diff
@@ -135,92 +137,138 @@ export const EndpointEntry: FunctionComponent<EndpointEntryProps> = ({
           entityLookup: features.entityLookup?.location,
         },
       });
+
+      showOverlay(<AddResolverConfirmation />);
     } catch (e) {
       onSetAllError(e.message);
     }
     setIsLoading(false);
   };
 
   return (
-    <tr className={className}>
-      <th>
+    <div
+      className={`flex flex-col rounded-xl shadow-lg w-80 mt-6 pt-12 pb-8 ${
+        orderNumber % 2 !== 0 ? "md:bg-cerulean-50" : "bg-white"
+      } ${
+        isEditable ? "h-130" : "h-auto"
+      } md:bg-none md:w-full md:h-auto md:rounded-none md:shadow-none md:mt-0 md:p-4`}
+      onMouseOver={() => setHoverState(true)}
+      onMouseOut={() => setHoverState(false)}
+    >
+      <div className="flex flex-col md:flex-row">
+        <div className={`hidden md:inline-block ${!isEditable ? "w-3.5 mr-3" : "ml-5"}`}>
+          {!isEditable && (
+            <div className={`text-xl text-cloud-300 ${hoverState ? "flex flex-col" : "hidden"}`}>
+              <i className="fas fa-sort-up hover:text-cloud-900" onClick={onMoveEntryUp} />
+              <i className="fas fa-sort-down -mt-4 hover:text-cloud-900" onClick={onMoveEntryDown} />
```

Comment:
ux wise, able hard to click (small area), mobile don't have move up move down eh? iirc this feature will resolve addresses accordingly to order priority, might be crucial to have

Reply:
ok done, increased the size of it and ya mobile dont have because its on hover then will appear. will have to check with jerry how should i go about including this.

Reply:
um show it upfront then?